lambdas/data-migration1/src/async-operations.ts

```diff
@@ -22,18 +21,31 @@ export const migrateAsyncOperationRecord = async (
   dynamoRecord: AWS.DynamoDB.DocumentClient.AttributeMap,
   knex: Knex
 ): Promise<void> => {
+  const asyncOperationPgModel = new AsyncOperationPgModel();
+
   // Use API model schema to validate record before processing
   Manager.recordIsValid(dynamoRecord, schemas.asyncOperation);
 
-  if (await doesRecordExist({ id: dynamoRecord.id }, knex, tableNames.asyncOperations)) {
+  let existingRecord;
+  try {
+    existingRecord = await asyncOperationPgModel.get(knex, { id: dynamoRecord.id });
+  } catch (error) {
+    // Swallow any RecordDoesNotExist errors and proceed with migration,
+    // otherwise re-throw the error
+    if (!(error instanceof RecordDoesNotExist)) {
+      throw error;
+    }
+  }
```

Comment:
We should probably replicate this logic across the other migrations so that they're using the `.get` method on the models, but we don't have to do it in this PR

Reply:
No problem, easy enough to do here:

https://github.com/nasa/cumulus/pull/2162/commits/5578e2474b3198a3b4640b1e81c74746198a6a42